dist/cityManager.js

```diff
@@ -0,0 +1,54 @@
+class CityManager {
+    constructor() {
+        this.cityData = []
+    }
+    getCityData = async function (cityName) {
+        let cityInfo = await $.get(`/city/${cityName}`)
+        this.cityData.unshift(cityInfo)
+        return cityInfo
+    }
+
+    getDataFromDB = async function () {
+        let citiesData = await $.get('/cities')
+        citiesData.forEach(c => this.cityData.push(c))
```

Comment:
Why do you need to do a loop here? You can just do:
`this.cityData = citiesData`